.editorconfig

```diff
@@ -0,0 +1,12 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
```suggestion
insert_final_newline = true

[Makefile]
indent_style = tab
indent_size = 4

```

Reply:
Oh indeed, I removed the Makefile part originally because I didn't want to make one. And yet, I ended up doing one :'(